src/Raven.Server/Web/Studio/LicenseHandler.cs

```diff
@@ -134,5 +135,44 @@ public async Task RenewLicense()
                 context.Write(writer, renewLicense.ToJson());
             }
         }
+
+        [RavenAction("/license-server/connectivity", "GET", AuthorizationStatus.ValidUser)]
+        public async Task CheckConnectivityToLicenseServer()
+        {
+            var result = new ConnectivityToLicenseServer();
+
+            try
+            {
+                var response = await ApiHttpClient.Instance.GetAsync("/Subscription.svc");
```

Comment:
You do not define a timeout which means it will take up to 100 seconds (default)

Reply:
I do not think that `/Subscription.svc` is an appropriate endpoint

First of all this is 3.5 endpoint, second this endpoint is not a lightweight endpoint. There should be a dedicated one e.g. `/alive` or sth like that

Reply:
The `/Subscription.svc` endpoint is lightweight (`GET` request) and was added for v3.5.
I'll add a new endpoint for this.

Reply:
done